tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -5866,7 +5886,7 @@ node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(8795,39): error
 node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(8799,38): error TS2339: Property 'offset' does not exist on type '{ [x: string]: any; node: any; start: number; end: number; collapse: any; coverStart: any; coverE...'.
   Property 'offset' does not exist on type '{ [x: string]: any; node: any; start: number; end: number; collapse: any; coverStart: any; coverE...'.
 node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(8817,16): error TS2345: Argument of type 'boolean' is not assignable to parameter of type 'number'.
-node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(8818,3): error TS2322: Type 'Timer' is not assignable to type 'boolean'.
+node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(8818,3): error TS2322: Type 'number' is not assignable to type 'boolean'.
```

Comment:
is this a result of this change, or just an update in the baselines?

Reply:
`node` re-typed `setTimeout` to return `Timer` instead of just `number`.